### ISSUE TITLE
Hard coded repositoryId and ServerPoolId?

### ISSUE THREAD

**Issue by royolsen** [dbgeek/ansible_oracle_cloud_modules#5]
I noticed that these values are hard coded in the oracle_ovm module. 

Am I correct in assuming that these should better be set correctly for the current OVM environment?

            "repositoryId": {"type": "com.oracle.ovm.mgr.ws.model.Repository",
                             "value": "0004fb00000300008564bbdbe309abfb"},
            "serverPoolId": {"type": "com.oracle.ovm.mgr.ws.model.ServerPool",
                             "value":"0004fb000002000047a9ab9d92c370b8"},

**Comment by dbgeek** [comment]
Yes, it should be changed to current ovm environment that you want to run against. But optimal it should be parameter into the ansible module. 
I will try to fix that..

If you have heard about terraform i have started to play around with a terraform provider for OVM
https://github.com/dbgeek/terraform-provider-ovm

**Comment by royolsen** [comment]
Well, they appear to only be used for the "createvm" state. I guess the "present" state is more commonly used. In the latter, I guess repository is set by the clone definition and the "serverPoolId" is correctly retrieved using name from "serverPool".

I have heard about terraform (by the Vagrant guys, right?), I have not yet had a chance to look at it.
